scripts/yarn/vets-website-assets.js

```diff
@@ -0,0 +1,171 @@
+  vagovdev: devBucket,
+  vagovstaging: stagingBucket,
+  vagovprod: prodBucket,
+}
+
+// This file exists in the above buckets and contains a list to all compiled assets that should be included.
+const fileManifestPath = 'generated/file-manifest.json'
+
+// Path to assets in a vets-website repo cloned adjacent to next-build.
+const vetsWebsiteAssetPath = '../vets-website/src/site/assets'
+
+const destinationPath = './public/generated/'
+
+// Function that loops through to download all compiled js assets listed in a bucket's manifest.
+async function downloadFromLiveBucket(buildtype) {
+  const bucket = BUILD_TYPE_BUCKET[buildtype]
+
+  const fileManifestRequest = await fetch(`${bucket}/${fileManifestPath}`)
+  const fileManifest = await fileManifestRequest.json()
+
+  const files = []
+
+  files[fileManifestPath] = {
+    path: fileManifestPath,
+    contents: Buffer.from(JSON.stringify(fileManifest)),
+  }
+
+  const entryNames = Object.keys(fileManifest)
+
+  const downloads = entryNames.map(async (entryName) => {
+    let bundleFileName = fileManifest[entryName]
+    const bundleUrl = bundleFileName.includes(bucket)
+      ? `${bundleFileName}`
+      : `${bucket}${bundleFileName}`
+    const bundleResponse = await fetch(bundleUrl)
+
+    if (bundleFileName.includes('generated/../')) {
+      console.log(`Excluding: ${bundleFileName} from download`)
+    } else {
+      if (!bundleResponse.ok) {
+        throw new Error(`Failed to download asset: ${bundleUrl}`)
+      }
+
+      if (bundleFileName.startsWith('/')) {
+        bundleFileName = bundleFileName.slice(1)
+      }
+
+      files[bundleFileName] = {
+        path: bundleFileName,
+        // No need to store file contents here since
+        // assets will be stored directly on disk
+        contents: '',
+      }
+
+      // Store file contents directly on disk
+      fs.outputFileSync(
+        path.join('./public/', bundleFileName),
+        await bundleResponse.buffer()
+      )
+    }
+  })
+  return Promise.all(downloads)
+}
+
+// Gather assets that are expected by the compiled files but not included in the bucket (because content-build would source them).
+// These are font files, icons, and other assorted images.
+async function moveAssetsFromVetsWebsite() {
+  console.log('Moving additional assets from adjacent vets-website repo...')
+
+  try {
+    fs.copySync(`${vetsWebsiteAssetPath}/fonts`, destinationPath)
+    console.log('Copied font files from vets-website.')
+
+    fs.copySync(`${vetsWebsiteAssetPath}/img`, './public/img/')
+    console.log('Copied image assets from vets-website.')
+
+    // Some stylesheets from vets-website expect these additional font files, but they are not included
+    // in the bucket files or in that repo's font folder. We source them directly from the node module.
+    fs.copySync(
+      './node_modules/@fortawesome/fontawesome-free/webfonts',
+      destinationPath,
+      { errorOnExist: false, force: true, dereference: true }
+    )
```

Comment:
This feels interesting to me. It seems odd that we'd copy something directly from `node_modules`. Doesn't vets-website include this as a dependency. How is it being utilized in the current `content-build/vets-website` setup?

Reply:
the fontawesome files are not included in the bucket we download assets from. `content-build` actually includes the [fontawesome icons directly in the repo](https://github.com/department-of-veterans-affairs/content-build/tree/main/src/site/assets/img/icons), which may be something we want to look into in the future, but isn't currently compatible with how we are currently sourcing our assets. i am not quite sure how the paths resolve correctly for that vs including the font files directly, to be entirely clear.